packages/syntax-core/src/WordConfetti/WordConfetti.test.tsx

```diff
@@ -26,4 +26,22 @@ describe("wordConfetti", () => {
     expect(screen.getByText("test")).toBeInTheDocument();
     expect(screen.getByText("word")).toBeInTheDocument();
   });
+
+  it("returns a consistent color and tilt for words", () => {
```

Comment:
@christianvuerings What's the best way to test rotation and background color? I had to add a data-testid to `WordConfetto` but right now it's just the text itself which isn't unique.

Also not sure if the uses of `toMatchInlineSnapshot` are how we want to do this

Reply:
@jliotta see https://github.com/Cambly/syntax/pull/559 for an example. Yes, it's possible to use `toMatchInlineSnapshot` but we should then avoid adding the padding values as well